powershell/tools/qltest.cmd

```diff
@@ -1,9 +1,11 @@
 @echo off
 
-if not defined CODEQL_POWERSHELL_EXTRACTOR (
-    set CODEQL_POWERSHELL_EXTRACTOR=Semmle.Extraction.PowerShell.Standalone.exe
-)
-
-type NUL && "%CODEQL_EXTRACTOR_POWERSHELL_ROOT%/tools/%CODEQL_PLATFORM%/%CODEQL_POWERSHELL_EXTRACTOR%"
+type NUL && "%CODEQL_DIST%\codeql.exe" database index-files ^
+    --prune=**/*.testproj ^
+    --include-extension=.ps1 ^
+    --size-limit=5m ^
+    --language=powershell ^
+    --working-dir=. ^
+    "%CODEQL_EXTRACTOR_POWERSHELL_WIP_DATABASE%"
```

Comment:
Doesn't this need to be a list of files? I guess the CI works so I must be missing something

Reply:
Note that this calls `codeql database index-files`. That doesn't take a list of files. That command will ultimately end up calling the `index-files` script with an argument that is a path to a file containing a newline-separated list of files to index.